core/block/editor/state/state.go

```diff
@@ -1378,6 +1385,35 @@ func (s *State) IsEmpty(checkTitle bool) bool {
 	return true
 }
 
+// isEmptyDataview is a separate emptiness check for sets and collections, as
```

Comment:
as.....?🤔

Reply:
Sorry, started typing and did not finish :smile: